### PR TITLE
Waiv 1396

### PR DESCRIPTION
**Pull request by oksanana**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

**Comment by oksanana** [comment]
1396, 1264, 1431, 1365

File: src/client/object/WobjHeader.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ const WobjHeader = ({
   const parentName = wobject.parent
     ? getClientWObj(wobject.parent, usedLocale)[objectFields.name]
     : '';
+  const objectTypes = useRef(['list', 'page']).current;
```
Comment (milklim) [inline src/client/object/WobjHeader.js]: do not use ref in this case

File: src/client/object/WobjHeader.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { useContext } from 'react';
+import React, { useContext, useRef } from 'react';
```
Comment (milklim) [inline src/client/object/WobjHeader.js]: remove ref import

File: src/client/object/WobjHeader.js [hunk 3]
```diff
@@ -45,6 +46,15 @@ const WobjHeader = ({
     </div>
   );
 
+  const getLink = () => {
+    const link = `/object/${wobject.author_permlink}`;
+    if (isEditMode) return null;
+    if (isMobile) return `${link}/about`;
+    if (wobject && objectTypes.includes(wobject.object_type))
```
Comment (milklim) [inline src/client/object/WobjHeader.js]: check object_type with values from objectType const 
(https://github.com/Waiviogit/waivio/blob/dev/src/client/object/const/objectTypes.js)

**Pull request merged**